spec/systems/signup_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
@malparty How about changing the naming to `authenticated`/ `unauthenticated` everywhere instead of `non logged`/ `logged`